src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs

```diff
@@ -199,6 +211,17 @@ static int EncodeHttpInteger(long longToEncode, Span<byte> buffer)
             }
         }
 
+        public async Task<int> ReadDataStreamAsync(byte[] payload)
+        {
+            int bytesRead = await _stream.ReadAsync(payload.AsMemory(0)).ConfigureAwait(false);
```

Comment:
@CarnaViire  The definition of the function uses "Memory<byte> buffer" and I just wanted my variable to be the same as in the function, but it also works without
But since I will remove it entirely this will also be removed